pywinauto/controls/uia_controls.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (C) 2016 airelil
+# Copyright (C) 2015 Intel Corporation
```

Comment:
UIA/multi-backend features development is my personal activity, so Intel copyright is not necessary in the new files.